Directory.Packages.props

```diff
@@ -50,6 +50,7 @@
     System.TypeInitializationException : The type initializer for 'Tmds.Utils.ExecFunction' threw an exception.
     System.NotSupportedException : Application is running as testhost, unable to determine parent 'dotnet' process.
     -->
+    <PackageVersion Include="System.Text.Json" Version="7.0.3" />
```

Comment:
I think this isn't needed in this PR yet?

Reply:
Removed `PackageVersion`. This was required before I removed the unnecessary `PrivateAssets="All"`.